custom/icds_reports/migrations/sql_templates/database_views/child_health_monthly.sql

```diff
@@ -0,0 +1,113 @@
+CREATE TABLE IF NOT EXISTS public."config_report_icds-cas_static-child_health_cases_a46c129f"
```

Comment:
I think you should be able to get rid of this now correct? You may also need to remove https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/tests/__init__.py#L97